Makefile

```diff
@@ -254,6 +254,10 @@ hosts-firefox:
 	            ln -sfv "$(LIB_DIR)/browserpass/hosts/firefox/$(APP_ID).json" "/usr/lib/mozilla/native-messaging-hosts/$(APP_ID).json"; \
 	            [ -e "/usr/lib/mozilla/native-messaging-hosts/$(APP_ID).json" ] || echo "Error: the symlink points to a non-existent location" >&2; \
 	            ;; \
+	FreeBSD)    mkdir -p "$(LIB_DIR)/mozilla/native-messaging-hosts/"; \
+	            ln -sfv "$(LIB_DIR)/browserpass/hosts/firefox/$(APP_ID).json" "${LIB_DIR}/mozilla/native-messaging-hosts/$(APP_ID).json"; \
+	            [ -e "${LIB_DIR}/mozilla/native-messaging-hosts/$(APP_ID).json" ] || echo "Error: the symlink points to a non-existent location" >&2; \
+	            ;; \
```

Comment:
Thanks! Since it seems the code is exactly the same as for `Linux)`, let's instead combine them, I believe this should work: `Linux|FreeBSD)`